### DISCUSSION TITLE
32 bit aax or rtas for macos.

### DISCUSSION THREAD

**Discussion by SOK-888** [apohl79/audiogridder#744] [Q&A] [answered]
Hi there, you have made a very genius project that takes audio processing to another level!
I just want to ask you if it is easy to make a 32 bit aax or rtas version for mac ? I am using Pro Tools 10
on a 10.9.5 system.Thank's in advance!

**Comment by CaBuffalo** [comment, marked as answer]
Sorry, 32bit will not be supported.

On Wed, Oct 6, 2021 at 6:31 AM SOK-888 ***@***.***> wrote:

> Hi there, you have made a very genius project that takes audio processing
> to another level!
> I just want to ask you if it is easy to make a 32 bit aax or rtas version
> for mac ? I am using Pro Tools 10
> on a 10.9.5 system.Thank's in advance!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/744>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BKSID3LXC33F542BDUFRFTNANCNFSM5FOS7RQQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by SOK-888** [reply]
That's ok, thank you for your response!
